core/client/app/routes/setup/one.js

```diff
@@ -41,6 +41,11 @@ export default Ember.Route.extend({
         return DownloadCountPoller.create({url: this.get('ghostPaths.count')});
     },
 
+    setupController: function (controller) {
+        this._super(controller);
+        this.controllerFor('setup').send('toStepOne');
+    },
```

Comment:
Yeah I think with the other changes you and kevinansfield have suggested I don't need this kind of backward logic anyway. Learning a lot about Ember on this PR it's awesome